pkg/kind/kind.go

```diff
@@ -168,6 +168,16 @@ func checkForExistingCluster() error {
 			}
 		} else {
 			fmt.Println("\n    Installation skipped")
+			checkKnativeNamespace := exec.Command("kubectl", "get", "namespaces")
+			output, err := checkKnativeNamespace.CombinedOutput()
+			namespaces := string(output)
+			if err != nil {
+				fmt.Println(string(output))
+				return fmt.Errorf("check existing cluster: %w", err)
+			}
+			if strings.Contains(namespaces, "knative") {
+				return fmt.Errorf("knative installation already exists, aborting")
```

Comment:
If we reach this scenario, let's first give the user a chance to delete/recreate the cluster. And if they decline, then we can say something like "Knative already installed, ending quickstart" (feel free to tweak the phrasing) and exit. We also probably want to shutdown gracefully in this scenario, as it wouldn't be an error.